Essentials/src/com/earth2me/essentials/commands/Commandtpahere.java

```diff
@@ -1,13 +1,16 @@
 package com.earth2me.essentials.commands;
 
-import com.earth2me.essentials.User;
-import org.bukkit.Server;
+
+import static com.earth2me.essentials.I18n.tl;
 
 import java.util.Collections;
 import java.util.List;
 
-import static com.earth2me.essentials.I18n.tl;
+import org.bukkit.Server;
+
+import com.earth2me.essentials.User;
```

Comment:
Any particular reason to reorder the imports, or was this just an IDE thing?

Reply:
IDE thing, I'll fix it if you like :)